server/server.js

```diff
@@ -18,9 +19,24 @@ const writeEnvironmentVariablesToFile = () => {
 const basePath = '/rekrutteringsbistand-stilling';
 const buildPath = path.join(__dirname, 'build');
 
+const setupProxy = (fraPath, tilTarget) =>
+    createProxyMiddleware(fraPath, {
+        target: tilTarget,
+        changeOrigin: true,
+        secure: true,
+        pathRewrite: (path) => {
+            const nyPath = path.replace(fraPath, '');
+            console.log(`~> Proxy fra '${path}' til '${tilTarget + nyPath}'`);
```

Comment:
Er det greit å beholde denne loggingen? Vi har vel ikke flere fnr i URL-en nå, og den er ganske nyttig for debugging og sånt

Reply:
Fjerne polø.